clang/test/CodeGenSYCL/kernel-param-acc-array-ih.cpp

```diff
@@ -49,4 +57,4 @@ int main() {
   Accessor acc[2];
 
   a_kernel<class kernel_A>([=]() { acc[1].use(); });
-}
+}
```

Comment:
Somehow this comment disappeared?  Newline still needed here.

Also, changes on 35/40 should probably go away unless clang-format made you do it.